.github/workflows/publish-docker-ecr.yml

```
@@ -0,0 +1,278 @@
+            platform: linux/amd64
+            runner: ubuntu-latest
+            arch: amd64
+            gate: langwatch-gateway
+          - image: langwatch-gateway
+            dockerfile: Dockerfile.go_service
+            platform: linux/arm64/v8
+            runner: ubuntu-24.04-arm
+            arch: arm64
+            gate: langwatch-gateway
+    runs-on: ${{ matrix.runner }}
+    if: needs.changes.outputs[matrix.gate] == 'true'
```

actionlint (1.7.12)

[error] context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)